pkg/test/framework/components/environment/kube/flags.go

```diff
@@ -369,11 +280,11 @@ func normalizeFile(originalPath string) (string, error) {
 func init() {
 	flag.StringVar(&kubeConfigs, "istio.test.kube.config", "",
 		"A comma-separated list of paths to kube config files for cluster environments.")
-	flag.BoolVar(&settingsFromCommandLine.Minikube, "istio.test.kube.minikube", settingsFromCommandLine.Minikube,
```

Comment:
Did you intentionally unexport the variable?

Reply:
Yes, it's deprecated in favor of the LoadBalancerSupported, the only use of this outside this package was a `if Minikube { err } `, so I just moved that in-package.